persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -75,6 +75,7 @@ class HmdaFileValidator(submissionId: String) extends HmdaPersistentActor with L
         .filter(x => x.isInstanceOf[LarParsed])
         .map(e => e.asInstanceOf[LarParsed].lar)
         .map(lar => validateLar(lar, ctx).toEither)
+        .map { e => println(e); e }
```

Comment:
I assume this is just a debugging statement. Should it maybe be removed or changed to `log.debug`?